armbian/base/customize-armbian-rockpro64.sh

```diff
@@ -476,7 +491,7 @@ redis-cli SET bitcoind:version "${BITCOIN_VERSION}"
 
 # LIGHTNING --------------------------------------------------------------------
 BIN_DEPS_TAG="v0.0.1-alpha"
-LIGHTNING_VERSION_BUILD="0.7.2.1"
+LIGHTNING_VERSION_BUILD="0.7.3"
```

Comment:
Collides with https://github.com/digitalbitbox/bitbox-base/pull/256/commits/35bec85bcb9dc5f2a0ac52e7d1f298776bef642a from #256.  I'd prefer to not have it in here and only in #256.

Reply:
Had to include it here as version 0.7.2.1  fails to build on the RockPro64 directly. Will merge https://github.com/digitalbitbox/bitbox-base/pull/256 first, remove that change, rebase and then merge this pull request.